instaclustr/resource_cluster.go

```diff
@@ -1160,7 +1160,7 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	for _, dataCentre := range cluster.DataCentres {
 		for _, node := range dataCentre.Nodes {
 			if !stringInSlice(node.Rack, azList) {
-				if !strings.HasPrefix(node.Size, "zk-") {
+				if !strings.HasPrefix(node.Size, "zk-") && !strings.HasPrefix(node.Size, "KDZ-") {
```

Comment:
i know these were already there, but there are now 3 duplicate and would be good to put it in a single place (and possibly comment why its there)